pkg/jx/cmd/step_collect_integration_test.go

```diff
@@ -0,0 +1,35 @@
+package cmd_test
```

Comment:
If this is an integration test, you need to add in the file header this comment:
```
// +build integration
```